stylix/palette.nix

```
@@ -10,7 +10,10 @@ let
 
   paletteJSON = let
     generatedJSON = pkgs.runCommand "palette.json" { } ''
-      ${palette-generator}/bin/palette-generator ${cfg.polarity} ${cfg.image} $out
+      ${palette-generator}/bin/palette-generator \
+        "${cfg.polarity}" \
+        "${lib.escapeShellArg "${cfg.image}"}" \
```

The external quotation marks here are not necessary since `lib.escapeShellArg` adds them.

```suggestion
        ${lib.escapeShellArg "${cfg.image}"} \
```